docs/docs/porting-part-of-a-html-site-to-gatsby.md

```diff
@@ -521,11 +521,48 @@ website-domain
 {/* highlight-end */}
 ```
 
+## Porting more parts
+
+To replace multiple parts of a website with a single Gatsby application there are a few additional steps:
+
+1. Mirror the website structure within the Gatsby `/src/pages` folder.
+
+2. Adjust the `pathPrefix` Gatsby configuration option to reflect the new root path at which the Gatsby application will be served.
+
+3. Ensure the `to` props on all Gatsby `<Link>` components are correct relative to `/src/pages/` as the application root.
+
+For the example website covered in this guide, here is what each step involves for also migrating the `services` part of the site:
+
+1. Create a `/src/pages/who` folder, and move into it all the files currently in `/src/pages`. Create a new `/src/pages/services` folder and follow the html migration steps above for the `/services/` HTML files. The already completed Layout component will save a lot of the work!
+
+```
+gatsby-site-sections/src/pages
+
+├── who
+│   ├── index.js
+│   ├── ellla-arborist.js
+│   ├── marin-leafer.js
+│   └── sam-surgeon.js
+└── services
+    ├── index.js
+    ├── growing.js
+    ├── cleaning.js
+    └── shrinking.js
+```
+
+2. The `pathPrefix` configuration option is no longer needed as the Gatsby application will now be served at the root of the site alongside the remaining HTML files.
```

Comment:
but we still need the build with  `--withPaths` for the assets?

Reply:
Aye, there's a note up with the build step on line 479 that I think does the trick in explaining this:

`> **Note**: The `--prefix-paths` option _must_ be used for path and asset prefixes to be applied`

Thinking they'll find themselves back up there when building again if they hit problems.

Reply:
Here's a question: what does the asset prefix stuff gain? Could this guide be simplified and include global CSS instead?

Reply:
Looking again at the [asset prefix docs](https://www.gatsbyjs.org/docs/asset-prefix/) page as well, I think it'll follow on from this guide really cleanly for the reader. I'll take a pass through to take out asset-prefix from the example, host the whole gatsby build output at the /who path and move the css, png etc to assets folder on the website-domain.

Reply:
What do you mean by global CSS? Directly importing a CSS file in the Layout.js file? I'd lean more toward keeping a basic link tag so the method stays consistent with the remaining HTML pages, so

`<link
      rel="stylesheet"
      type="text/css"
      href="https://gatsby-html-partial-assets.s3.eu-west-2.amazonaws.com/style.css"
    />`

becomes

`<link rel="stylesheet" type="text/css" href="/assets/style.css"/>`